src/main/cpp/sim/run_stride.cpp

```diff
@@ -31,6 +32,10 @@
 #include <string>
 #include <utility>
 
+std::atomic<bool> stride::util::INTERRUPT(false);
+std::atomic<unsigned int> stride::util::INTERVAL(1);
```

Comment:
I think `INTERVAL` should really be a member of the configuration, not a global variable.